packages/account-v2/src/utils/connectedAppsUtils.ts

```diff
@@ -0,0 +1,15 @@
+type TConnectedAppsHeader = {
+    [key: string]: string;
+};
+
+export const connectedAppsHeader: TConnectedAppsHeader = {
+    action: 'Action',
```

Comment:
Can we restructure this as as an array of object as required by the table component. That way u needn't have an array and an object and use a map function in the component to generate it

Reply:
Would that be a better approach?

Reply:
I found that we were using similar approach for displaying columns in `loginHistory` component which is already in master